website/docs/d/aviatrix_firenet.html.markdown

```diff
@@ -2,12 +2,14 @@
 layout: "aviatrix"
 page_title: "Aviatrix: aviatrix_firenet"
 description: |-
-  Gets the Aviatrix FireNet.
+  Gets an Aviatrix FireNet's details.
 ---
 
 # aviatrix_firenet
 
-Use this data source to get the Aviatrix firenet for use in other resources.
+The **aviatrix_firenet** data source provides details about a specific FireNet created by the Aviatrix Controller.
+
+This resource can prove useful when a module accepts a FireNet's detail as an input variable.
```

Comment:
"This data source"

Reply:
I used the formatting you see because I wanted to keep consistent formatting across all docs, similar to our resource markdowns. What do you think?

By avoiding "This data source", and substituting with bolded text, it makes it clear what the sentence that proceeds it talks about. Formatting was modeled after AWS's.

Reply:
I mean in line 12, "This resource" should be "This data source"

Reply:
Ah I see now. Will change